include/network/uri/uri_builder.hpp

```diff
@@ -245,7 +261,7 @@ class uri_builder {
   void set_port(string_type port);
   void set_authority(string_type authority);
   void set_path(string_type path);
-  void append_query(string_type query);
+  void append_query_parameter(string_type query);
   void append_query_key_value_pair(string_type key, string_type value);
```

Comment:
For consistency reasons, what about renaming the public and private `append_query_key_value_pair(key, value)` to `append_query_parameter(name, value)`?

Reply:
Because I already went through this bike shed discussion when I first named it :) It's not part of the original PR so I won't change it here.

Reply:
I agree